package.json

```diff
@@ -38,8 +38,8 @@
     "@iarna/toml": "2.2.5",
     "@prettier/is-es5-identifier-name": "0.2.0",
     "@prettier/parse-srcset": "3.1.0",
-    "@typescript-eslint/typescript-estree": "6.3.0",
-    "@typescript-eslint/visitor-keys": "6.3.0",
+    "@typescript-eslint/typescript-estree": "6.4.0",
+    "@typescript-eslint/visitor-keys": "6.4.0",
```

Comment:
Can you upgrade `@typescript-eslint/eslint-plugin` too? So the dependencies doesn't get duplicated.

Reply:
done [5cdb8a6](https://github.com/prettier/prettier/pull/15249/commits/5cdb8a6ae8eeffc5af6600ceb0d29f6e0bee1f74)